addons/base_vat/base_vat.py

```diff
@@ -359,4 +363,51 @@ def check_vat_tr(self, vat):
 
         return False
 
-# vim:expandtab:smartindent:tabstop=4:softtabstop=4:shiftwidth=4:
+    def check_vat_nl(self, vat):
+        """
+        Temporary Netherlands VAT validation to support the new format
```

Comment:
Is this the commit that you cherry picked? I'm surprised that the comment is different: https://github.com/odoo/odoo/commit/3ddaf382cb353829bcd226f114abcf4c8e0f0157#diff-5818ba1e95e0716e509401478e1aacc8R387-R392